src/components/Dashboard/supply_markets/usdcSupplyMarketDialog.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import TabPanel from "@mui/lab/TabPanel";
+
+// Dialogs
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+
+// Token Information
+import usdcTokenLogo from "../../../assets/img/usdc_coin_token.png";
+import {
+  Box,
+  Divider,
+  IconButton,
+  Stack,
+  Tab,
+  Typography,
+} from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+
+import USDCBorrowDetails from "../widgets/usdcBorrow";
+import SupplyDetails from "../widgets/supply/supplyDetails";
+import WithdrawDetails from "../widgets/withdraw/withdrawDetails";
+
+// Action Items
+import { useDispatch, useSelector } from "react-redux";
+import { AppDispatch, RootState } from "../../../app/Store";
+import EnableWarning from "../widgets/enableWarning";
+import { updateSupplyBalance, updateUSDCSupplyRate, updateUSDCBalance } from "../../../features/dashboard/USDCMarketSlice";
+import { useEffect } from "react";
+import { updateBorrowLimit } from "../../../features/dashboard/AccountSlice";
+import { Transition } from "../../../utils/Transition";
+
+interface SupplyMarketDialogProps {
+  open: boolean;
+  onClose: () => void;
+  title: string;
+}
+
+function USDCSupplyMarketDialog(props: SupplyMarketDialogProps) {
+  const [value, setValue] = React.useState("0");
```

Comment:
[Comment]

Same here with the value being a type string

Reply:
supply dialog uses tabs, which only accept string types not number types